python/tvm/driver/tvmc/frontends.py

```diff
@@ -88,10 +88,76 @@ def import_keras():
         from tensorflow import keras
 
         return tf, keras
+    except ImportError:
+        err = (
+            "Error: Tensorflow is required and was not found.\n"
+            'Please install it using "pip install tensorflow".'
+        )
+        raise TVMCException(str(err))
     finally:
         sys.stderr = stderr
 
 
+def import_onnx():
+    """Lazy import function for onnx"""
+    try:
+        # pylint: disable=C0415
+        import onnx as _onnx
+    except ImportError:
+        err = (
+            "Error: ONNX is required and was not found.\n"
+            'Please install it using "pip install onnx".'
+        )
+        raise TVMCException(str(err))
+
+    return _onnx
+
+
+def import_tensorflow():
+    """Lazy import function for tensorflow"""
+    try:
+        # pylint: disable=C0415
+        import tensorflow as tf
+    except ImportError:
+        err = (
+            "Error: Tensorflow is required and was not found.\n"
+            'Please install it using "pip install tensorflow".'
+        )
+        raise TVMCException(str(err))
+
+    return tf
+
+
+def import_torch():
+    """Lazy import function for torch"""
+    try:
+        # pylint: disable=C0415
+        import torch as tc
+    except ImportError:
+        err = (
+            "Error: Torch is required and was not found.\n"
+            'Please install it using "pip install torch".'
+        )
+        raise TVMCException(str(err))
+
+    return tc
+
+
+def import_tflite():
+    """Lazy import function for tflite.Model"""
+    try:
+        # pylint: disable=C0415
+        import tflite.Model as model
+    except ImportError:
+        err = (
+            "Error: tflite is required and was not found.\n"
+            'Please install it using "pip install tflite".'
+        )
+        raise TVMCException(str(err))
```

Comment:
These seem to follow a similar pattern, could this be moved to a function? Something like:
```py
def create_import_error_string(library_name, import_name):
    err = (
        f"Error: {library_name} is required and was not found.\n"
        f'Please install it using "pip install {import_name}".'
    )
    return str(err)
```
And used as:
```suggestion
        raise TVMCException(create_import_error_string('tflite', 'tflite'))
```